packages/Brew/yugabyte.rb

```diff
@@ -0,0 +1,55 @@
+class Yugabyte < Formula
+  desc "High-performance distributed SQL database Yugabyte DB"
+  homepage "https://yugabyte.com"
+  url "https://downloads.yugabyte.com/yugabyte-2.0.3.0-darwin.tar.gz"
+  sha256 "5d7d35b3bbdee7b89fe7910a363734b559cd5ce7e5899cd155024fd965fd8145"
+
+  def install
+    rm_f buildpath/"lib/cassandra-driver-internal-only-3.13.0.post0-743d942c.zip"
+    rm_f buildpath/"lib/futures-2.1.6-py2.py3-none-any.zip"
+    rm_f buildpath/"lib/six-1.7.3-py2.py3-none-any.zip"
+    prefix.install Dir["*"]
+    bin.install_symlink prefix/"yugabyte-db"
+    bin.install_symlink prefix/"bin/ysqlsh"
+    bin.install_symlink prefix/"bin/ycqlsh"
+  end
+
+  plist_options :manual => "yugabyte-db start"
```

Comment:
yugabyte-db was renamed to yugabyted in the latest package

Reply:
updated in brew formula